src/pages/research.astro

```diff
@@ -0,0 +1,1170 @@
+        "26th International Conference on Software Engineering (ICSE 2004)",
+      location: "Edinburgh, Scotland",
+      publisher: "IEEE Computer Society",
+      pages: "583–592",
+    },
+    preprintOrArchiveUrl:
+      "https://nixos.org/~eelco/pubs/immdsd-icse2004-final.pdf",
+  },
+  {
+    title: "Integrating Software Construction and Software Deployment",
+    authors: [{ name: "Eelco Dolstra" }],
+    year: 2003,
+    abstract:
+      "Classically, software deployment is a process consisting of building the software, packaging it for distribution, and installing it at the target site. This approach has two problems. First, a package must be annotated with dependency information and other meta-data. This to some extent overlaps with component dependencies used in the build process. Second, the same source system can often be built into an often very large number of variants. The distributor must decide which element(s) of the variant space will be packaged, reducing the flexibility for the receiver of the package. In this paper we show how building and deployment can be integrated into a single formalism. We describe a build manager called Maak that can handle deployment through a sufficiently general module system. Through the sharing of generated files, a source distribution transparently turns into a binary distribution, removing the dichotomy between these two modes of deployment. In addition, the creation and deployment of variants becomes easy through the use of a simple functional language as the build formalism.",
+    publicationInfo: {
+      type: "conference",
+      conference:
+        "11th International Workshop on Software Configuration Management (SCM-11)",
+      location: "Portland, Oregon, USA",
+      pages: "102–117",
+    },
+    preprintOrArchiveUrl: "https://nixos.org/~eelco/pubs/iscsd-scm11-final.pdf",
+  },
+];
+
+const sponsors = await getEntry("sponsors", "info");
+
+const papersByYear = papers.reduce((acc, paper) => {
+  validatePaper(paper);
+  const year = paper.year;
+  if (!acc[year]) acc[year] = [];
+  acc[year].push(paper);
+  return acc;
+}, {});
+
+const sortedYears = Object.keys(papersByYear).sort((a, b) => b - a);
+
+function getVenueString(pubInfo: PublicationType) {
+  switch (pubInfo.type) {
+    case "conference":
+      return `${pubInfo.conference}, ${pubInfo.location}`;
+    case "journal":
+      return `${pubInfo.journal}${pubInfo.volume ? `, vol. ${pubInfo.volume}` : ""}${pubInfo.number ? `, no. ${pubInfo.number}` : ""}`;
+    case "thesis":
+      return `${pubInfo.thesisType} thesis, ${pubInfo.institution}, ${pubInfo.location}`;
+  }
+}
+---
+
+<Layout title="Research and Scientific Publications">
+  <PageHeader text="Research and Scientific Publications" />
+  <Container class="grid px-12 pt-16">
+    <p class="text-xl font-bold text-gray-500">
+      Nix is a research project who's time to solve today's software
+      distribution issues has come.
+    </p>
+  </Container>
```

Comment:
```suggestion
```

Semantically this would have been an `<aside>` btw. In any case I wouldn't want any of this to sound like advertisement but instead do the topic of the page justice and present things as a matter of fact.